fixtures/vcr_cassettes/synopsis.yaml

```
@@ -0,0 +1,113 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Connection:
+      - close
+      Host:
+      - api.trello.com
+      User-Agent:
+      - Python-urllib/3.12
+    method: GET
+    uri: https://api.trello.com/1/boards/REDACTED/lists?key=REDACTED&token=REDACTED&cards=open
+  response:
+    body:
+      string: '[{"id":"REDACTED","name":"To Do","closed":false,"color":null,"idBoard":"REDACTED","pos":16384,"subscribed":false,"softLimit":null,"cards":[{"id":"REDACTED","badges":{"attachmentsByType":{"trello":{"board":0,"card":0}},"location":false,"votes":0,"viewingMemberVoted":false,"subscribed":false,"fogbugz":"","checkItems":0,"checkItemsChecked":0,"checkItemsEarliestDue":null,"comments":0,"attachments":0,"description":false,"due":null,"dueComplete":false,"start":null},"checkItemStates":[],"closed":false,"dueComplete":false,"dateLastActivity":"2024-03-15T15:11:05.958Z","desc":"","descData":{"emoji":{}},"due":null,"dueReminder":null,"email":null,"idBoard":"REDACTED","idChecklists":[],"idList":"REDACTED","idMembers":[],"idMembersVoted":[],"idShort":REDACTED,"idAttachmentCover":null,"labels":[],"idLabels":[],"manualCoverAttachment":false,"name":"Test
```

Not sure if you backed out of this, but I can't convince myself this is working! I'll leave a couple of suggestions where I think things aren't quite lined up below, but there's some missing quotes in the redacting here that causes me some issues to start:
```suggestion
      string: '[{"id":"REDACTED","name":"To Do","closed":false,"color":null,"idBoard":"REDACTED","pos":16384,"subscribed":false,"softLimit":null,"cards":[{"id":"REDACTED","badges":{"attachmentsByType":{"trello":{"board":0,"card":0}},"location":false,"votes":0,"viewingMemberVoted":false,"subscribed":false,"fogbugz":"","checkItems":0,"checkItemsChecked":0,"checkItemsEarliestDue":null,"comments":0,"attachments":0,"description":false,"due":null,"dueComplete":false,"start":null},"checkItemStates":[],"closed":false,"dueComplete":false,"dateLastActivity":"2024-03-15T15:11:05.958Z","desc":"","descData":{"emoji":{}},"due":null,"dueReminder":null,"email":null,"idBoard":"REDACTED","idChecklists":[],"idList":"REDACTED","idMembers":[],"idMembersVoted":[],"idShort":"REDACTED","idAttachmentCover":null,"labels":[],"idLabels":[],"manualCoverAttachment":false,"name":"Test
```